tests/src/JIT/Methodical/Arrays/misc/_relgcarr.csproj

```diff
@@ -27,7 +27,7 @@
     </CodeAnalysisDependentAssemblyPaths>
   </ItemGroup>
   <PropertyGroup>
-    <DebugType></DebugType>
+    <DebugType>Full</DebugType>
```

Comment:
I don't think that this is correct: looking at the original build files, the rel build of this test should not have debug info enabled (i.e. `DebugType` should remain unset).